rclcpp_lifecycle/test/test_lifecycle_node.cpp

```diff
@@ -437,6 +492,12 @@ TEST_F(TestDefaultStateMachine, check_parameters) {
   test_node->set_parameters({int_parameter});
```

Comment:
May I ask, what's the diference between `set_parameter` and `set_parameters`?

Reply:
`set_parameter` sets only a single parameter, but `set_parameters` takes a collection, which is why `int_parameter` is enclosed in `{}`. `set_parameters_atomically` will set all parameters, but only if they all succeed.